recipes/python-statuspage/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set url_version = "0.3-pre" %}
+{% set version = url_version.replace("-", "") %}
+
+package:
+  name: python-statuspage
```

Comment:
I prefixed this. I don't what our status is on prefixing these days. That being said, if we decide to try a different status page in the future, I wanted to keep the space open for alternatives.